src/app/public/modules/i18n/resources.service.ts

```diff
@@ -32,26 +32,24 @@ import {
   SkyAppLocaleProvider
 } from './locale-provider';
 
-const defaultResources: {[key: string]: {message: string}} = {};
+declare type SkyResourceType = {[key: string]: {message: string}};
```

Comment:
Would consumers need to reference this type?

Reply:
No. It's only used locally to this file. Because of the strong typing I get from `HttpClient`, that type needed to be cited more than once, so I defined it as a type instead of copying the type literal everywhere.